### PR TITLE
pkg/lokomotive: make NewCluster to not return error

### PR DESCRIPTION
**Pull request by invidian**
As we never do that anyway.

Signed-off-by: Mateusz Gozdek <mateusz@kinvolk.io>

### CODE REVIEW

**Review by knrt10 — DISMISSED** [review]
+1

File: pkg/lokomotive/cluster.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ type Cluster struct {
 	ExpectedNodes int
 }
 
+// NewCluster constructs and returns new Cluster object.
```
Comment (iaguis) [inline pkg/lokomotive/cluster.go:34]:
```suggestion
// NewCluster constructs and returns a new Cluster object.
```

Comment (invidian) [reply pkg/lokomotive/cluster.go]: Thanks, fixed.

**Review by iaguis — APPROVED** [review]

**Review by knrt10 — DISMISSED** [review]
Hoping for AKS to pass too. LGTM

**Review by iaguis — DISMISSED** [review]

**Review by knrt10 — APPROVED** [review]
+1

**Pull request merged**